fili-core/src/test/groovy/com/yahoo/bard/webservice/data/config/table/BaseTableLoaderSpec.groovy

```diff
@@ -111,19 +102,19 @@ class BaseTableLoaderSpec extends Specification {
         metricNames = TestDruidMetricName.getByLogicalTable(SHAPES)
         physDefs = TestPhysicalTableDefinitionUtils.buildShapeTableDefinitions(new TestDimensions(), metricNames)
         dimNames = TestApiDimensionName.getByLogicalTable(SHAPES)
-        tableNames = physDefs.collect {it.name} as Set
+        tableNames = physDefs.name as Set
```

Comment:
Groovy allows shortcutting the spread-dot operator on collections for property access of elements in the collection. So, in this case, these are equal:

```groovy
physDefs*.name == physDefs.name
```

The result is a `SpreadCollection`, that is being coerced into a `Set`